python/ray/ray_constants.py

```diff
@@ -151,7 +151,7 @@ def to_memory_units(memory_bytes, round_up):
                                              30)
 
 # The reporter will report its' statistics this often (milliseconds).
```

Comment:
Minor grammar change: while we're updating this part of the code, can we use `its` instead of `its'`?